source/sync/data-access-patterns/flexible-sync.txt

```diff
@@ -212,3 +213,50 @@ sync objects with the backend {+app+} and can watch for and react to changes.
 
       To create queries from the .NET Client SDK, see the :ref:`.NET SDK 
       guide to Flexible Sync <dotnet-flexible-sync>`.
+
+Optimize Sync with Asymmetric Sync
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Asymmetric Sync is a sync strategy that is optimized for applications 
+with heavy client-side *insert-only* workloads. You enable it for one or more 
+collections. It supports writing to any collection type, including an 
+{+atlas-short+} time-series collection. 
+
+.. note::
+
+   Asymmetric Sync is a feature of Flexible Sync and cannot be enabled on apps 
+   that use Partition-Based Sync.
+
+For example, an IoT app that frequently logs sensor data has a significant 
+write workload, and essentially no read workload. The device may also be offline 
```

Comment:
```suggestion
write workload, and no read workload. The device may also be offline 
```
let's get rid of the word essentially here, since asymmetric sync doesn't support querying.